src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/ClaimIssuedNotificationService.java

```diff
@@ -47,9 +47,10 @@ public void sendMail(
         final String targetEmail,
         final Optional<String> pin,
         final String emailTemplateId,
-        final String reference
+        final String reference,
+        final Optional<String> claimantName
```

Comment:
Do not use `Optional` as input parameter: https://dzone.com/articles/optional-anti-patterns

Reply:
actioned